v2/azure-arm.yaml

```
@@ -934,6 +939,23 @@ objectModelConfiguration:
       Factory:
         $export: true
         $supportedFrom: v2.1.0
+  dataprotection: 
+    2023-01-01: 
+      BackupVault: 
+        $export: true 
+        $supportedFrom: v2.2.0 
+      BackupVaults_BackupInstance: 
+        $exportAs: BackupVaultsBackupInstance 
```

Suggest simplifying the name:

```suggestion
        $exportAs: BackupInstance 
```

Everything is already uniquely namespaced, so we can simplify the names without causing problems.

---

Tried to rectify the same as mentioned, but is hows error. Posting it over Teams to look over.